bc/channel/utils/connectors/threads_api/client.py

```diff
@@ -0,0 +1,174 @@
+import logging
+import time
+import uuid
+
+import requests
+from django.conf import settings
+
+from bc.core.utils.images import convert_to_jpeg, resize_image
+from bc.core.utils.s3 import put_object_in_bucket
+
+logger = logging.getLogger(__name__)
+
+
+_BASE_API_URL = "https://graph.threads.net/v1.0"
+
+
+class ThreadsAPI:
```

Comment:
All get and post requests must have timeouts: https://github.com/psf/requests/issues/3070